lib/loader/index.d.ts

```diff
@@ -1,12 +1,12 @@
 import "@types/webassembly-js-api";
 
 /** WebAssembly imports with two levels of nesting. */
-interface ImportsObject {
-  [key: string]: {},
-  env: {
+interface ImportsObject extends Record<string, any> {
+  env?: {
     memory?: WebAssembly.Memory,
     table?: WebAssembly.Table,
-    abort?: (msg: number, file: number, line: number, column: number) => void
+    abort?: (msg: number, file: number, line: number, column: number) => void,
+    trace?: (msg: number, n: string, ...rest: any[]) => void
```

Comment:
```ts
trace?: (msg: number, numArgs?: number, ...rest: any[]) => void
```

Reply:
second arg should be optional

Reply:
Fixed in 5b5e7be with co-authorization 👍

Reply:
thanks!